frontend/tests/features/cellGuide/cellGuide.test.ts

```diff
@@ -819,32 +819,40 @@ describe("Cell Guide", () => {
         );
 
         const navbar = page.getByTestId(CELL_GUIDE_CARD_NAVIGATION_SIDEBAR);
-
+        const sourceData = page.getByTestId(CELL_GUIDE_CARD_SOURCE_DATA_TABLE);
+        const ontologyView = page.getByTestId(
+          CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW
+        );
         // scroll to the bottom
+
         const section3 = page.getByTestId("section-3");
         await section3.scrollIntoViewIfNeeded();
-
-        // check that source data is in viewport
-        const sourceData = page.getByTestId(CELL_GUIDE_CARD_SOURCE_DATA_TABLE);
-        await sourceData.waitFor({ timeout: WAIT_FOR_TIMEOUT_MS });
-        expect(sourceData).toBeInViewport();
-
+        await tryUntil(
+          async () => {
+            // check that source data is in viewport
+            expect(sourceData).toBeInViewport();
+            // 1 second in between retries or we hit the retry limit too fast
+            await page.waitForTimeout(1000);
```

Comment:
TryUntil accepts an option to increase retry limit, so we don't have to hard wait this 1 second!

Reply:
I'd prefer to wait for 1 second so I don't have to guess how many retries I should increase to (which is functionally equivalent to guessing a wait time, just more abstract!). The `expect` line runs really fast.

Reply:
Ah that's a good point. Maybe the API should change to maxTimeour instead of retries. Will update on Monday;